pkg/secrets/filestore.go

```diff
@@ -0,0 +1,15 @@
+package secrets
+
+import (
+	v1 "k8s.io/api/core/v1"
+)
+
+const FilestoreSecretType string = "secrets.kanister.io/filestore"
```

Comment:
Lets not export the const unless necessary 
```suggestion
const filestoreSecretType string = "secrets.kanister.io/filestore"
```

Reply:
We would need them while writing integration tests, else we will have to redefine them again over there, hence keeping them exported